spinalcordtoolbox/scripts/sct_run_batch.py

```diff
@@ -306,6 +388,10 @@ def run_single(subj_dir, script, script_args, path_segmanual, path_data, path_da
     })
     if 'SCT_DIR' not in envir:
         envir['SCT_DIR'] = __sct_dir__
+    if include_files:
+        envir['INCLUDE_FILES'] = ' '.join(include_files)
+    if exclude_files:
+        envir['EXCLUDE_FILES'] = ' '.join(exclude_files)
```

Comment:
A shell purist would tell you that this is unsafe for filenames which could contain whitespace, but:
- such filenames are widely known to cause problems, and
- the usual advice to delimit filenames with null bytes doesn't apply, because environment variables cannot contain null bytes.

So, this is fine.